charts/agent/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
```

Comment:
I assume we will have several more revisions of this chart - I think roles will need some switches base on other configurations if we are taking a least privilege approach.